app/_layout.tsx

```diff
@@ -9,14 +9,15 @@ const RootLayout = () => {
       <SafeAreaView
         style={{ backgroundColor: colors.white, padding: ScaleSize(10) }}
       />
-      <Stack screenOptions={{ headerShown: false }}>
+      <Stack screenOptions={{ headerShown: true }}>
```

Comment:
I like everything except this part. Please change it back to false

Reply:
Then we get stuck on the invitation page and can't go back

Reply:
You can swipe on the left side of the page to go back to org page, since you used a router.push. Also some considerations to styling of it (for me it can be changed after), it could be sliding on the invitation with deletion on the left side and accept on the right side.